docs/v1/P4Runtime-Spec.adoc

```diff
@@ -1991,24 +1813,24 @@ value_sets {
    field, except for the `@match` annotation, if present (see the `match` field
    below).
  * `bitwidth`: set to the value of `W` for the corresponding struct field.
- * `type_name`, which indicates whether the struct field has a [user-defined
-   type](#sec-user-defined-types); this is useful for
-   [translation](#sec-psa-metadata-translation).
+ * `type_name`, which indicates whether the struct field has a xref:sec-user-defined-types[user-defined type]; this is useful for xref:sec-psa-metadata-translation[translation].
  * `match`: by default `match_type` is set to `EXACT`; the P4 programmer can
    specify a different match type by using the `@match` annotation
-   [@P4SelectExpr].
+   cite:[P4SelectExpr].
  * `doc`: documentation associated with the struct field.
```

Comment:
Nit - the PDF rendering has a line break after the bullet, see bottom of page 34. `doc` appears on page 35

Reply:
Yep, I saw that, too.  Seems like a bug in asciidoctor-pdf to me.  I tried re-generating PDF from the latest version of this PR, and it isn't like this, possibly because that bullet item is in the middle of a page, instead of at the end.